.github/workflows/build.yml

```
@@ -0,0 +1,94 @@
+name: Build
+
+on:
+    push:
+        branches: [main, next, master, 2.5, 3.4, 4.3, 5.1, 6.2]
+    pull_request:
+    release:
+        types: [created]
+
+jobs:
+    tests:
+        runs-on: ubuntu-latest
+        name: Build and test
+        strategy:
+            matrix:
+                php: [7.3, 7.4]
+                composer-flags: [""]
+                composer-extras: [""]
+                stability: ["stable"]
+                include:
+                    -   php: 7.3
+                        publish-phar: true
+                    -   php: 7.3
+                        composer-flags: --prefer-lowest
+                    -   php: 7.3
+                        composer-extras: "symfony/event-dispatcher ^3.4 symfony/contracts ^2.0"
+                    -   php: 7.4
+                        stability: "dev"
+
+        env:
+            COMPOSER_ROOT_VERSION: dev-master
+
+        steps:
+            -   uses: actions/checkout@v2
+
+            -   name: Setup PHP
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: "${{ matrix.php }}"
+                    ini-values: "phar.readonly=0"
+
+            -   name: Install composer extras
+                if: matrix.composer-extras != ''
+                run: composer require ${{ matrix.composer-extras }} --no-update
+
+            -   name: Update the minimum stability
+                if: matrix.stability == 'dev'
+                run: composer config minimum-stability ${{ matrix.stability }}
+
+            -   name: Install dependencies
+                run: composer update ${{ matrix.composer-flags }}
+
+            -   name: Run tests (phpspec)
+                run: bin/phpspec run --format=dot
+
+            -   name: Run tests (phpunit)
+                run: ./vendor/bin/phpunit -v
+
+            -   name: Run tests (behat)
+                if: false
```

How about replacing this with

```suggestion
                continue-on-error: true
```

---

This should go away once behat tests are fixed.

---

Fair enough!